### PR TITLE
docs(core): fix broken links and add missing latest prefix

### PR DESCRIPTION
**Pull request by meeroslav**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: scripts/documentation/generate-executors-data.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ function generateTemplate(
     Options can be configured in \`${filename}\` when defining the executor, or when invoking it.
     ${
       framework != 'angular'
-        ? `Read more about how to use executors and the CLI here: https://nx.dev/${framework}/getting-started/nx-cli#running-tasks.`
+        ? `Read more about how to use executors and the CLI here: https://nx.dev/latest/${framework}/getting-started/nx-cli#common-commands.`
```
Comment (bcabanes) [inline scripts/documentation/generate-executors-data.ts:90]: I would remove the "latest/${framework}" to have a generic url instead https://nx.dev/getting-started/nx-cli#common-commands.

File: packages/workspace/src/generators/workspace/files/README.md__tmpl__ [hunk 2]
```diff
@@ -12,9 +12,9 @@ This project was generated using [Nx](https://nx.dev).
 
 [Nx Documentation](https://nx.dev/angular)
 
-[10-minute video showing all Nx features](https://nx.dev/angular/getting-started/what-is-nx)
+[10-minute video showing all Nx features](https://nx.dev/latest/angular/getting-started/intro)
```
Comment (bcabanes) [inline packages/workspace/src/generators/workspace/files/README.md__tmpl__:15]: Same here https://nx.dev/getting-started/intro

Comment (meeroslav) [reply packages/workspace/src/generators/workspace/files/README.md__tmpl__:15]: I've changed it for the both links

File: scripts/documentation/generate-executors-data.ts (hunk 1 above)
Comment (meeroslav) [reply scripts/documentation/generate-executors-data.ts]: FYI, I had to update `internal-link-checker` to allow missing framework in the link.

**Review by bcabanes — APPROVED** [review]

**Pull request merged**